fix: implement changes keeping metatags (#1078)
aziontech/docs: src/content/docs/en/pages/architectures/edge-application/image-processing.mdx

---
title: Optimize and process images with Azion Edge Platform

> description: Explore Azion Image Processor for efficient Edge Computing solutions.
>   Automate resource management, optimize images, and streamline content delivery with
>   advanced cache policies.

Reviewer: ```suggestion
description: Explore Azion Image Processor to automate resource management, optimize images, and streamline content delivery with advanced cache policies.
```
